feat(content): Add log entries for Incipias (#10357)
endless-sky/endless-sky: data/incipias/incipias first contact.txt

			`	The voice now takes on a calm tone. "Your persistence on this topic is admirable, but allow me to explain my conviction with an example."`
			label care
			action

> 				log `Entered a region of space northeast of the Hai. Was warned by a disembodied voice to not disturb the 'unborn flowers' that inhabit that region.`

Reviewer: ```suggestion
				log `Entered a region of space northeast of the Hai and was warned by a disembodied voice to not disturb the 'unborn flowers' that inhabit the region.`
```
Logs are a bit less "Point. Thing. Done." and more "The point, with the thing, and done," if that makes sense.
Also "that" makes more sense to be "the" since you're already referencing the region, not pointing at it as "that".

				event "label incipias space"
			`	"Why do humans care about art?" The voice now seems far away yet more present than ever. "Because it amazes you, the same way it amazes us to see another kind grow and overcome their burdens. The lives of others are like a painting that never stops being painted, but is a tragic thing to watch the painting stop. That is why we try to protect these creations of life - so we can see what heights they reach."`
			`	There is a pause. "I see that you can not be convinced by me. Though I could intervene in your journey, I do not favor unsavory confrontation." The voice is now resolute in tone. "I will let you pass, but I will be watching you, as I am the guardian of this space."`
			`	The voice fades out leaving nothing behind but the feeling of being watched.`
				decline
			label agree
			action

> 				log `Entered a region of space northeast of the Hai. Was warned by a disembodied voice to not disturb the 'unborn flowers' that inhabit that region.`

Reviewer 2: ```suggestion
				log `Entered a region of space northeast of the Hai. Was warned by a disembodied voice to not disturb the 'unborn flowers' that inhabit that region. Agreed to stay away from there.`
```

> 				log `Entered a region of space northeast of the Hai. Was warned by a disembodied voice to not disturb the 'unborn flowers' that inhabit that region. Agreed to stay away from there.`

Reviewer: ```suggestion
				log `Entered a region of space northeast of the Hai and was warned by a disembodied voice to not disturb the 'unborn flowers' that inhabit the region. Agreed to stay away from there.`
```

		to despawn
			has "incipias: helped the derelict"
		on assist

> 			log `Assisted a stranded alien vessel in an uninhabited system, whose inhabitants appeared similar to floating octopi or jellyfish. Could not communicate with them due to the language barrier, but managed to give them the repair equipment they needed to repair their ship. However, they lacked a hyperdrive and so were unable to travel back to their home system.`

Reviewer: ```suggestion
			log `Assisted a stranded alien vessel in an uninhabited system with crew members that appear similar to floating octopi or jellyfish. Could not communicate with them due to the language barrier, but managed to give them the repair equipment they needed to repair their ship. However, they lacked a hyperdrive and so were unable to travel back to their home system.`
```
This is a better example of more written-out logs. Though, having "inhabitants" read right after "uninhabited" is a bit jarring. Changed it so it specifically refers to the crew you met.

> 			log `Assisted a stranded alien vessel in an uninhabited system with crew members that appear similar to floating octopi or jellyfish. Could not communicate with them due to the language barrier, but managed to give them the repair equipment they needed to repair their ship. However, they lacked a hyperdrive and so were unable to travel back to their home system.`

Reviewer: I'm assuming this is supposed to say, "appeared to be"? "Whose crew members appeared to floating jellyfish" isn't grammatically correct.

I'm not sure if we need to explicitly avoid octopus words. Even if there are multiple uses, and eventually someone might think it's wrong, that's not really reason to completely throw out the reference when it's an important descriptor. If someone thinks it's wrong, they'll likely look it up and find out that there are multiple references, so at that point it's up to the author which word to use, not whoever thinks it's wrong/not reason to completely remove it.

> 			log `Assisted a stranded alien vessel in an uninhabited system with crew members that appear similar to floating octopi or jellyfish. Could not communicate with them due to the language barrier, but managed to give them the repair equipment they needed to repair their ship. However, they lacked a hyperdrive and so were unable to travel back to their home system.`

Reviewer 2: It is also part of who they are, not something to be just removed because the word is difficult.

> 			log `Assisted a stranded alien vessel in an uninhabited system with crew members that appear similar to floating octopi or jellyfish. Could not communicate with them due to the language barrier, but managed to give them the repair equipment they needed to repair their ship. However, they lacked a hyperdrive and so were unable to travel back to their home system.`

Reviewer 3: Should have been `appeared similar to`. I accidentally removed that word.

I understand the importance of the octopus descriptor, but this also seems to be the first time it's used in the Incipias content ("Incipias: The Derelict" uses "octopod" instead). So, is the fact that they look like octopuses the important descriptor, or is the important factor that they're octopedal?

Here's my suggestion without the accidental change and keeping the octopi:
```suggestion
			log `Assisted a stranded alien vessel in an uninhabited system whose crew members appeared similar to floating octopi or jellyfish. Could not communicate with them due to the language barrier, but managed to give them the equipment they needed to repair their ship. Unfortunately, they lacked a hyperdrive and so were unable to travel back to their home system.`
```

				`	"The risks do sound concerning. I will leave them alone after all."`
					goto declined
			label yes
				set "language: Incipias"

> 				log `Discovered that the species to the northeast of Hai space, the Incipias, are under the protection of the Quarg. Was given a translator so the stranded Incipias ship could be assisted.`

Author: I'm pretty sure that first and second person pronouns aren't supposed to be used in log entries.

> 				log `Discovered that the species to the northeast of Hai space, the Incipias, are under the protection of the Quarg. Was given a translator so the stranded Incipias ship could be assisted.`

Reviewer 3: Maybe this then?
```
log `Discovered that the species to the northeast of Hai space, the Incipias, are under the protection of the Quarg. Was given a translator in order to assist a stranded Incipias ship.`
```

				`	"I want to help one of your stranded ships."`
			`	Again a long pause. "Your intention-goals are welcome. Please dock to the capital-station." Along the message comes a small map of the station with the docking ports highlighted.`
	on complete
		log `Met with the Incipias on their space station, and gained access to their ships, which are capable of entering the gas giant that the stranded Incipias ship has taken refuge in.`

> 		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system and are under the protection of the Quarg.`

Reviewer 2: ```suggestion
		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated from a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system and are under the protection of the Quarg.`
```
from feels right, on not, but I might be stupid

				`	"I want to help one of your stranded ships."`
			`	Again a long pause. "Your intention-goals are welcome. Please dock to the capital-station." Along the message comes a small map of the station with the docking ports highlighted.`
	on complete

> 		log `Met with the Incipias on their space station. Gained access to their ships, which are capable of entering the gas giant that the stranded Incipias ship has taken refuge in.`

Reviewer: ```suggestion
		log `Met with the Incipias on their space station, and gained access to their ships, which are capable of entering the gas giant that the stranded Incipias ship has taken refuge in.`
```

> 		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system. They are under the protection of the Drak and Quarg, and was warned not to give them any advanced technology or interfere in their development.`

Reviewer 4: ```suggestion
		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system. They are under the protection of the Drak and Quarg, and I was warned not to give them any advanced technology or interfere in their development.`
```

Author: Logs don't use first person pronouns.

Reviewer 4: ```suggestion
		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system. They are under the protection of the Drak and Quarg, warning against giving them any advanced technology or interfering with their development.`
```

Author: How about this?
```suggestion
		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system. They are under the protection of the Drak and Quarg, who warned against giving them any advanced technology or interfering with their development.`
```

> 		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system. They are under the protection of the Drak and Quarg, and was warned not to give them any advanced technology or interfere in their development.`

Reviewer: ```suggestion
		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system. Learned that they are under the protection of the Quarg, and was warned not to give them any advanced technology or interfere in their development.`
```
We don't know it's the Drak (even if we've been in Sayiaban, since we don't ever see it), the only way you would know is if you were hostile with them, which at that point there's nothing further in the area anyways.

> 		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system. Learned that they are under the protection of the Quarg, and was warned not to give them any advanced technology or interfere in their development.`

Reviewer: I think it makes sense. Look at other faction logs; there are other details in there as well. This log only comes after gaining information enough about the Incipias, not initially after a short conversation about them/you meeting them similar to other facrtions.

> 		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system. Learned that they are under the protection of the Quarg, and was warned not to give them any advanced technology or interfere in their development.`

Reviewer 3: I guess my comment was unclear.

Other than the octopus thing and a little bit more evidence on the warning, I don't think my suggestion changes any content. Instead, it fixes the grammatical error in the last sentence, where it starts telling you what _you've_ done rather than who the Incipias are and what _they've_ done—something that I haven't seen in any other faction logs.

That said, here's a version without the octopus change:
```suggestion
		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system. They are under the protection of the Quarg, who have strictly warned you not to give them any advanced technology or interfere in their development.`
```

> 		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system. Learned that they are under the protection of the Quarg, and was warned not to give them any advanced technology or interfere in their development.`

Reviewer: With roadrunner's above comment - I'd keep the "Learned" in there if we don't do first/second person; other dialogs use this specific phrasing too ("Learned...").

> 		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system. Learned that they are under the protection of the Quarg, and was warned not to give them any advanced technology or interfere in their development.`

Author: This suggestion has the same problem as the one above. It would probably be easier if I just moved the last sentence to a regular log entry. How about:
```suggestion
		log "Factions" "Incipias" `The Incipias are a species of aliens that live to the northeast of Hai space. They originated on a gas giant, and resemble large, floating octopi. As they have not yet developed the hyperdrive, they are confined to a single system and are under the protection of the Quarg.`
```

			`	You tell them to board your ship, and shortly after, you take off. Luckily, you leave just in time to avoid a powerful windstorm approaching the area.`
				launch
	on complete

> 		log "Factions" "Incipias" `The Incipias appear to be split into the Hicemus and the Conlatio; the former occupies a space station, while the latter occupies the Incipias homeworld. The Conlatio do not appear to allow outsiders on their home planet.`

Reviewer: ```suggestion
		log "Factions" "Incipias" `The Incipias appear to be split into two groups called the Hicemus and the Conlatio; the former occupies a space station, while the latter occupies the Incipias homeworld. The Conlatio do not appear to allow outsiders on their home planet.`
```

> 		log "Factions" "Incipias" `The Incipias appear to be split into two groups called the Hicemus and the Conlatio; the former occupies a space station, while the latter occupies the Incipias homeworld. The Conlatio do not appear to allow outsiders on their home planet.`

Reviewer 3: Using "appear" twice is a bit stilted, although I'm not quite sure how to fix it.

Reviewer 5: ```suggestion
		log "Factions" "Incipias" `The Incipias seem to be split into two groups called the Hicemus and the Conlatio; the former occupies a space station, while the latter occupies the Incipias homeworld. The Conlatio do not appear to allow outsiders on their home planet.`
```
